source/includes/cli-guide/_cli-overview.html.md

```diff
@@ -1,3 +1,21 @@
 # CLI Overview
 
-The Bonsai Command Line Interface (CLI) is a text-based tool that enables you to configure and control the Bonsai Artificial Intelligence Engine. The CLI is especially useful for automation and connection to other tools. Currently, there are some actions that can only be performed using the CLI, such as loading your Inkling file and connecting your simulator. 
\ No newline at end of file
+```powershell
```

Comment:
It seems like a fair number of the shell instructions are the same in the windows and macOS directions.  What do you think about only giving macOS unless there is a difference, and explaining that in this first comment?

` This section is used when the Windows instructions are different from the macOS instructions.`